crates/shared/src/paraswap_api.rs

```diff
@@ -16,8 +16,6 @@ use {
     thiserror::Error,
 };
 
-const BASE_URL: &str = "https://apiv5.paraswap.io";
```

Comment:
I would keep this as:

```rust
const DEFAULT_URL: &str = "https://apiv5.paraswap.io";
```

Reply:
We also have a default value for the command line arg (which should be always used outside of tests). Should we then read/define the constant there or make the argument and Option?

Reply:
Sounds reasonable.

I don't feel strongly about my suggestion - it was more of a "note". Feel free to merge without it.